doc/api/http2.md

```diff
@@ -1627,6 +1627,7 @@ added: v8.4.0
   * `createConnection` {Function} An optional callback that receives the `URL`
     instance passed to `connect` and the `options` object, and returns any
     [`Duplex`][] stream that is to be used as the connection for this session.
+  * ...: Any [`net.connect()`][] or [`tls.connect()`][] options can be provided.
```

Comment:
Can you add backticks to the `...`.

Reply:
The current version matches `doc/api/tls.md` where it's used like this 3 times.